.github/workflows/main.yml

```diff
@@ -6,29 +6,63 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [ main, 'v*']
+    branches: [main, "v*"]
   pull_request:
-    branches: [ main, 'v*' ]
+    branches: [main, "v*"]
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
+  geotests:
```

Comment:
I'm thinking of the following reasons:
- We have special checks in the library itself (some modules are loaded and some modules not), 
- Some dependencies are installed for geo but not for usual: `binutils, gdal-bin, libproj-dev, libsqlite3-mod-spatialite`
- It would be better too to check if modules are properly processed in isolated environments